flake.nix

```diff
@@ -21,13 +21,13 @@
     };
 
     kitsune2 = {
-      url = "github:holochain/kitsune2?ref=v0.1.9";
+      url = "github:holochain/kitsune2?ref=v0.1.14";
```

Comment:
Shouldn't this be [0.1.15](https://github.com/holochain/holochain/blob/72826ef8d5e6999c27c2907d0afac385dbad9a14/crates/holochain_p2p/Cargo.toml#L44)?